lib/src/main/java/org/team100/lib/config/SimulatedCamera.java

```diff
@@ -0,0 +1,58 @@
+package org.team100.lib.config;
+
+import java.util.ArrayList;
+import java.util.Optional;
+
+import org.team100.lib.localization.PoseEstimationHelper;
+
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Rotation3d;
+import edu.wpi.first.math.geometry.Transform3d;
+import edu.wpi.first.math.geometry.Translation2d;
+
+public class SimulatedCamera {
+    private final Transform3d m_cameraInRobotCoordinates;
+
+    public SimulatedCamera(Transform3d cameraInRobotCoordinates) {
+        m_cameraInRobotCoordinates = cameraInRobotCoordinates;
+    }
+
+    //TODO make this work with any camera yaw
+    public Optional<ArrayList<Translation2d>> findNotes(Pose2d robotPose, Translation2d[] notes) {
```

Comment:
note this method isn't really "finding" anything, it's just transforming one planar geometry into another, so maybe call it something that reflects what it's actually doing?